authorship/vocalize_data.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import pandas as pd
+
+
+class VocalizeDataSource:
+    # specific on the vocalize output for frida
+    def __init__(self, voc_data, device='telephone'):
+        self._voc_data = voc_data
+        self._device = device
+
+    def get(self):
```

Comment:
Maybe a docstr here to describe what it returns?

Reply:
done